plugin.cfg

```diff
@@ -48,6 +48,7 @@ file:file
 auto:auto
 secondary:secondary
 etcd:etcd
+etcdv3:etcdv3
```

Comment:
better to call this etcd3 instead of etcdv3 while the latter is technically correct